webui/react/src/services/decoder.ts

```diff
@@ -209,7 +233,16 @@ export const jsonToExperimentDetails = (data: unknown): ExperimentDetails => {
     progress: ioType.progress !== null ? ioType.progress : undefined,
     startTime: ioType.start_time,
     state: ioType.state as RunState,
-    trials: ioType.trials.map(ioTrialToTrial),
+    trials: ioType.trials.map(ioTrial => {
+      /*
+       * `num_batches` is the number of batches for the trial ran for.
+       * We also need the running tally to show where in sequence the
+       * trial ran for, so we internally tally up the batches.
```

Comment:
nit: these two parts are confusing to me " batches for the trial ran for." "show where in sequence the trial ran for," let's do a quick zoom?

Reply:
yeah `numBatchesTally` wasn't needed, I was confused about this part, removed it!